framework/src/variables/MooseVariableData.C

```diff
@@ -1330,11 +1330,6 @@ MooseVariableData<OutputType>::computeAD(const unsigned int num_dofs, const unsi
   mooseAssert(_var.kind() == Moose::VarKindType::VAR_AUXILIARY || ad_offset || !_var_num,
               "Either this is the zeroth variable or we should have an offset");
 
-  // Hopefully this problem can go away at some point
-  if (ad_offset + num_dofs > AD_MAX_DOFS_PER_ELEM)
-    mooseError("Current number of dofs per element is greater than AD_MAX_DOFS_PER_ELEM of ",
```

Comment:
Why is this? even if it's in 3D with a bunch of variables? 81 will always be the most?

Reply:
thats fine, as long as it is an obvious error so its easy to debug if 81 is not enough.